.github/workflows/get-previous-release/action.yml

```diff
@@ -13,8 +17,13 @@ runs:
       run: |
         # If the channel is either "cloud" or "beta", fetch all Git tags matching the channel pattern,
```

Comment:
@davseve Very nice one liners

I'd suggest that we open a task to extract this script into a bash script that accepts arguments to make it easier to follow what's happening in the action.

Reply:
The oneliners are a mess, they are not so readable.
There is a lot we can refactor here...

Reply:
I think we can extract and refactor them in the next round as we add in weekly releases